lib/Doctrine/ORM/Query/Exec/AbstractSqlExecutor.php

```diff
@@ -10,6 +10,7 @@
 /**
  * Base class for SQL statement executors.
  * @todo Rename: AbstractSQLExecutor
+ * ???
```

Comment:
What's wrong here?

Reply:
Seems like the class was already renamed. May we can drop the comment?

Reply:
I don't think it was, the name is same, but casing is different `Sql` vs. `SQL`, hasn't been touched since 2009.